docs/source/tutorials_source/package/tutorial_custom_augmentations.py

```diff
@@ -380,4 +441,6 @@ def plot_knn_multilabels(embeddings, multilabels, samples_idx, n_neighbors=50):
 # plot the distribution of the multilabels of the k nearest neighbors of 
 # the three example images at index 4111, 3340, 1796
 k = 20
-plot_knn_multilabels(embeddings, multilabels, [4111, 3340, 1796], n_neighbors=k)
+plot_knn_multilabels(
+    embeddings, multilabels, [4111, 3340, 1796], fnames, n_neighbors=k
+)
```

Comment:
Cited from the tutorial:
_[We want to] evaluate how good the learned representations really are._

But after the plots there is no description/evaluation at all. I know this was missing in the tutorial already before, but it would be nice as a reader to follow the interpretation/evaluation of the author to find out whether the augmentations are good or bad.

Reply:
I agree. We should maybe also consider changing the evaluation in the first place. It is not very easy to understand.

Reply:
Ok thanks!